### PR TITLE
release 0.10.5

### PR DESCRIPTION
**Pull request by dkent600**
release 0.10.5

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,9 +1,15 @@
 ## 0.10.5
   - Features Added
-    - Recommended node.js version to build Alchemy is set to 12.16.1.
+    - change home page to show DAOs again instead of feed. Show separate "Your DAOs" section containing DAOs the user follows or is a member of, then the list of other DAOs below.
+    - added support for embedded youtube and vimeo videos in proposal descriptions
+    - recommended node.js version to build Alchemy is set to 12.16.2
+    - upgrade to use subgraph v39_3
```
Comment (orenyodfat) [inline CHANGELOG.md:6]: #1619 need to be merged to support subgraph v39_3. please merge it

Comment (dkent600) [reply CHANGELOG.md:6]: done

**Comment by EzraWeller** [comment]
This is still failing CI -- going to start reviewing b/c I've been told it's ready, but I hope it doesn't still need any significant changes!

**Comment by dkent600** [comment]
> This is still failing CI -- going to start reviewing b/c I've been told it's ready, but I hope it doesn't still need any significant changes!

@EzraWeller I am seeing it passing

image

**Comment by EzraWeller** [comment]
Issues to fix before release (in progress):
#1623

**Pull request merged**